docs/framework.md

```diff
@@ -143,3 +144,22 @@ class MyApplication {
 ```
 
 If you are using League\Container (recommended), then you may simply add and share your own classes to the same container.  If you are using some other DI container, then you should use [delegate lookup](https://github.com/container-interop/fig-standards/blob/master/proposed/container.md#14-additional-feature-delegate-lookup) to combine them.
+
+## Using a Custom Configuration Loader
+
+Robo provides a very simple configuration loader. If you wish to use more capable loader, you may opt to do so. Replace the call to `Robo::createConfiguration()` with code similar to the following:
+```
+use Robo\Config\Config;
+use Robo\Config\YamlConfigLoader;
+use Robo\Config\ConfigProcessor;
+
+$config = new Config();
+$loader = new YamlConfigLoader();
+$processor = new ConfigProcessor();
+$processor->extend($loader->load('defaults.yml'));
+$processor->extend($loader->load('myconf.yml'));
```

Comment:
I think there may be a missing step here. At this point, $processor->unprocessedConfig will have two rows. How do those get processes and imported into $config?